pkg/sensors/tracing/kprobe_process_credentials_test.go

```diff
@@ -337,3 +346,129 @@ spec:
 	err = jsonchecker.JsonTestCheck(t, checker)
 	require.NoError(t, err)
 }
+
+func TestKprobeMatchCurrentCredValidity(t *testing.T) {
+	testutils.CaptureLog(t, logger.GetLogger())
+	ctx, cancel := context.WithTimeout(context.Background(), tus.Conf().CmdWaitTime)
+	defer cancel()
+
+	option.Config.HubbleLib = tus.Conf().TetragonLib
+	tus.LoadInitialSensor(t)
+	tus.LoadSensor(t, testsensor.GetTestSensor())
+	sm := tuo.GetTestSensorManager(t)
+
+	ruid := []v1alpha1.CredIDValues{
+		{
+			Operator: "Equal",
+			Values:   []string{"0:0"},
+		},
+	}
```

Comment:
This test is hard to read.
I would suggest an array of test cases, each with the filter and the expected result to make it easier to read and understand.